pkg/cmd/cli/cmd/create/deploymentconfig.go

```diff
@@ -0,0 +1,141 @@
+package create
+
+import (
+	"fmt"
+	"io"
+
+	"github.com/spf13/cobra"
+
+	kapi "k8s.io/kubernetes/pkg/api"
+	"k8s.io/kubernetes/pkg/api/meta"
+	cmdutil "k8s.io/kubernetes/pkg/kubectl/cmd/util"
+	"k8s.io/kubernetes/pkg/runtime"
+
+	"github.com/openshift/origin/pkg/client"
+	"github.com/openshift/origin/pkg/cmd/util/clientcmd"
+	deployapi "github.com/openshift/origin/pkg/deploy/api"
+)
+
+const (
+	DeploymentConfigRecommendedName = "deploymentconfig"
+
+	deploymentConfigLong = `
+Create a deployment config that uses a given image.`
```

Comment:
Describe what deployment configs are and why I want them.  Crib from the API doc if necessary.